react/daily-art-prompt/src/store/rootReducer.js

```diff
@@ -61,6 +61,80 @@ export const promptImagesReducer = (state = promptsImages, action) => {
                 }
             })
 
+        case TYPES.EDIT_COMMENT_CLASS_NAMES:
+            return state.map(image => {
+                if (image.id === action.payload.imageId) {
+                    const updatedCommentsClassName = image.comments.map(comment => {
+                        if (comment.id === action.payload.commentId) {
+                            return {
+                                ...comment,
+                                editing: true
+                            }
+                        }
+                        else {
+                            return comment
+                        }
+                    })
+                    return {
+                        ...image,
+                        comments: updatedCommentsClassName
+                    }
+                }
+                else {
+                    return image
+                }
+            })
+
+        case TYPES.CANCEL_EDIT_COMMENT_CLASS_NAMES:
```

Comment:
You could put both of these actions into one:

dispatch({ type: UPDATE_COMMENT_EDITING, payload: ...args, editing: true/false })